.rubocop.yml

```diff
@@ -3,11 +3,58 @@ AllCops:
   Exclude:
     - 'bundler/**/*'
     - 'lib/rubygems/resolver/molinillo/**/*'
+    # TODO: Remove all Exclude values after this line.
+    - 'test/**/*'
   TargetRubyVersion: 2.2
 
+Metrics/BlockLength:
```

Comment:
I usually disable all of the `Metrics` cops besides `LineLength`, fwiw, since I find them to just be noisy

Reply:
I'm also disabled code length of `Metrics`. It's not affecting to code quality.